src/graph/GoExecutor.cpp

```diff
@@ -267,17 +272,23 @@ Status GoExecutor::prepareOver() {
         }
 
         auto v = edgeStatus.value();
-        if (isReversely()) {
+        if (direction_ == OverClause::Direction::kForward) {
```

Comment:
This code is exactly the same as above, so it would be better to use a function to represent it.